src/vs/workbench/common/editor.ts

```diff
@@ -1350,7 +1351,7 @@ export enum EditorCloseMethod {
 	MOUSE
 }
 
-export function preventEditorClose(group: IEditorGroup, editor: EditorInput, method: EditorCloseMethod, configuration: IEditorPartConfiguration): boolean {
+export function preventEditorClose(group: IReadonlyEditorGroup, editor: EditorInput, method: EditorCloseMethod, configuration: IEditorPartConfiguration): boolean {
```

Comment:
To make this easier to use in all contexts, this could be an OR type: `group: IEditorGroup | IReadonlyEditorGroupModel`